### DISCUSSION TITLE
The X-axis is not incremented, and the Y-axis icons will all move every time they are refreshed

### DISCUSSION THREAD

**Discussion by gracewang9** [patrykandpatrick/vico#289] [Questions]
Hello! Two questions to ask: 1. The X-axis is not incremented, how can it be set to be incremented? 2. Every time the icon is refreshed, it will all move. Can it be partially refreshed? Refreshing only updates one piece of data, while other axes do not move?

https://github.com/patrykandpatrick/vico/assets/8500399/20407f0d-995b-4fe5-872f-0fb09b5827c3

**Comment by patrickmichalik** [comment]
Hello!

1. Could you elaborate on what you’re referring to?
2. `DefaultDiffProcessor` animates only those values that have changed. Here, it looks like the chart is being recreated. I’d suggest disabling the initial animation. Please see the `runInitialAnimation` parameter of the `Chart` composable, as well as `BaseChartView#runInitialAnimation`.

**Reply by gracewang9** [reply]
Thank you very much for your answer. Based on the ideas you provided, I have solved my problem

**Reply by patrickmichalik** [reply]
You’re welcome! I’m glad to hear that.
